src/main/default/lwc/lookup/lookup.js

```diff
@@ -264,13 +272,21 @@ export default class Lookup extends LightningElement {
     }
 
     handleRemoveSelectedItem(event) {
+        if (this.disabled) {
+            return;
+        }
+
         const recordId = event.currentTarget.name;
         this._curSelection = this._curSelection.filter((item) => item.id !== recordId);
         // Process selection update
         this.processSelectionUpdate(true);
     }
 
     handleClearSelection() {
+        if (this.disabled) {
```

Comment:
short question, what is the difference between the clear and the remove method?

Reply:
`handleRemoveSelectedItem` is used to remove multi-select items and `handleClearSelection` is used to remove single entry lookup selection